pipelines/toast_ground_sim.py

```diff
@@ -644,7 +689,7 @@ def main():
 
         pars[ 'temperature_only' ] = 'F'
         pars[ 'force_pol' ] = 'T'
-        pars[ 'kfirst' ] = 'T'
+        pars[ 'kfirst' ] = 'F' # 'T' DEBUG
```

Comment:
Just wanted to make sure that you intended to disable the destriping as the default...